library_api/repositories/book_repository.go

```diff
@@ -0,0 +1,28 @@
+package repositories
```

Comment:
I would have renamed the file to just book, because we already know that it is repository (in repository package)